packages/hoppscotch-common/src/components/environments/index.vue

```diff
@@ -296,7 +296,7 @@ watch(
 
 defineActionHandler("modals.environment.add", ({ envName, variableName }) => {
   editingVariableName.value = envName
-  if (variableName) editingVariableValue.value = variableName
```

Comment:
@nivedin, was this truthy check added for eliminating `null/undefined` values? I don't think there can be a case where we would end up with such values since we've control over the value supplied. Regardless, we could have a fallback to an empty string.

Reply:
it was added as a undefined/null blocker ig, I think empty string will work in this case